detect_variants/detect_variants.cwl

```diff
@@ -32,6 +32,24 @@ inputs:
         type: int?
     mutect_artifact_detection_mode:
         type: boolean?
+    mutect_max_alt_allele_in_normal_fraction:
+        type: float?
+    mutect_max_alt_alleles_in_normal_count:
+        type: int?
+    varscan_strand_filter:
+        type: int?
+        default: 0
```

Comment:
Does this workflow still work if we only store the defaults in `varscan/varscan_somatic.cwl` rather than have copies in each layer of the CWL?